pandas_gbq/gbq.py

```diff
@@ -926,28 +749,52 @@ def read_gbq(query, project_id=None, index_col=None, col_order=None,
         .. [console flow]
             http://google-auth-oauthlib.readthedocs.io/en/latest/reference/google_auth_oauthlib.flow.html#google_auth_oauthlib.flow.InstalledAppFlow.run_console
         .. versionadded:: 0.2.0
-
     dialect : {'legacy', 'standard'}, default 'legacy'
         'legacy' : Use BigQuery's legacy SQL dialect.
         'standard' : Use BigQuery's standard SQL (beta), which is
         compliant with the SQL 2011 standard. For more information
         see `BigQuery SQL Reference
-        <https://cloud.google.com/bigquery/sql-reference/>`__
-
-    **kwargs : Arbitrary keyword arguments
-        configuration (dict): query config parameters for job processing.
-        For example:
-
-            configuration = {'query': {'useQueryCache': False}}
-
-        For more information see `BigQuery SQL Reference
-        <https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#configuration.query>`__
+        <https://cloud.google.com/bigquery/sql-reference/>`
+    credentials: credentials object, default None (optional)
+        If generating credentials on your own, pass in. Otherwise, will attempt
+        to generate automatically
+    return_type: {'schema','list','df'}, default 'df'
+        schema returns an array of SchemaField objects, which you can access
+            `from pprint import pprint
+            [pprint(vars(field)) for field in schema]`
+        list returns a list of lists of the rows of the results; column names
+            are not included
+        df returns a dataframe by default
+    query_parameters: tuple (optional) Can only be used in Standard SQL
+        example: gbq.read_gbq("SELECT @param1 + @param2",
+                          query_parameters = (bigquery.ScalarQueryParameter(
+                                                      'param1', 'INT64', 1),
+                                              bigquery.ScalarQueryParameter(
+                                                      'param2', 'INT64', 2)))
+        <https://cloud.google.com/bigquery/docs/parameterized-queries>
+    configuration : dict (optional)
+        Because of current limitations <https://github.com/GoogleCloudPlatform/
+        google-cloud-python/issues/2765> only some configuration settings are
+        currently implemented. You can pass them along like in the following:
+        `read_gbq(q,configuration={'allow_large_results':True,
+                                   'maximum_billing_tier':2})`
+        Example allowable settings:
+            allow_large_results, create_disposition, default_dataset,
+            destination, flatten_results, priority, use_query_cache,
+            use_legacy_sql, dry_run, write_disposition, udf_resources,
+            maximum_billing_tier, maximum_bytes_billed
+            <http://google-cloud-python.readthedocs.io/en/latest/_modules/
```

Comment:
alot of this won't render. build the docs locally and you will see many sphinx errors

Reply:
I can research, but perhaps someone can save me time and point to a resource since I'm not familiar with what does and doesn't render documentation-wise.

Reply:
If I run `help(gbq.read_gbq)` or do shift+tab in Jupyter Notebook, the documentation seems to load fine. If you have an example of what's broken and the circumstance, happy to try and fix. Thank you!

Reply:
Link rendering syntax: http://www.sphinx-doc.org/en/stable/rest.html#external-links

Reply:
Thanks, I went through and fixed hyperlinks and cleaned it up a bit according to that doc. Tried to build locally from the existing docs folder. Console output appeared to show that Sphinx update ran successfully, but when I actually load the api html page, it still shows the existing documentation. I even deleted the html file, but it just rebuilt them with the existing documentation. So I can't actually verify that it works (perhaps someone else can?), but just eyeballing the docstring, it should be ok now.